src/GafferTest/ContextTest.cpp

```diff
@@ -238,7 +246,11 @@ std::tuple<int,int,int,int> GafferTest::countContextHash32Collisions( int contex
 		{
 			for( int j = 0; j < 20; j++ )
 			{
-				c.set( numberNames[j], rand_r( &rand_seed ) );
+#ifndef _MSC_VER
+					c.set( numberNames[j], rand_r( &rand_seed ) );
+#else
+					c.set( numberNames[j], rand() );
+#endif
```

Comment:
Can we just use a more recent random generator that works cross-platform? One from Imath or one of the new `std` ones? I'd like to limit the `#ifdefing` to the minimum necessary.

Reply:
fixed in 6beb287da180b823dac5a828d419c55cdc9fd282